onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -295,19 +333,37 @@ def destroy(self, request, *args, **kwargs):
         self.get_object().delete()
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    def _get_tableau_headers(self, xform, repeat_iters):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6116910)